package.json

```diff
@@ -52,13 +52,11 @@
     "ember-cli-uglify": "^1.2.0",
     "ember-export-application-global": "^1.0.5",
     "ember-gestures": "0.4.2",
-    "ember-getowner-polyfill": "1.1.1",
```

Comment:
I think this library is currently a dependency of `ember-gestures`. Are you sure this can be removed?